src/content/plugins/split-chunks-plugin.md

```diff
@@ -164,7 +165,11 @@ T> `maxSize` takes higher priority than `maxInitialRequest/maxAsyncRequests`. Ac
 
 `boolean: true | function (module, chunks, cacheGroupKey) | string`
 
-The name of the split chunk. Providing `true` will automatically generate a name based on chunks and cache group key. Providing a string or function will allow you to use a custom name. If the name matches an entry point name, the entry point will be removed.
+The name of the split chunk. Providing `true` will automatically generate a name based on chunks and cache group key.
+
+Providing a string or a function allows you to use a custom name. Specifying a string (or a function that always returns the same string) will merge all common modules and vendors into a single chunk. This probably leads to more code downloaded than needed.
```

Comment:
```suggestion
Providing a string or a function allows you to use a custom name. Specifying a string or a function that always returns the same string, will merge all common modules and vendors into a single chunk. This might lead to bigger initial downloads and slow down page's load.
```

Reply:
- Changed this to `Specifying either ... or ...` to make it clearer
- Used `slow down page loads` since it'll be more grammatically appropriate than `slow down page's load`.